controllers/shooter.py

```diff
@@ -43,6 +43,27 @@ def execute(self) -> None:
         tempoary replacement of magicbot statemachine
         """
         self.state()
+        self.update_LED()
+
+    def update_LED(self):
+        if self.shooter.is_at_speed() == True:
+            self.shooter.led_speed.setRGB(0, 255, 0)
```

Comment:
These should be calls to a method on the component, rather than accessing its internals.